public/app/features/scenes/core/SceneTimeRange.tsx

```diff
@@ -1,9 +1,17 @@
-import { TimeRange, UrlQueryMap } from '@grafana/data';
+import { getDefaultTimeRange, getTimeZone, TimeRange, UrlQueryMap } from '@grafana/data';
 
 import { SceneObjectBase } from './SceneObjectBase';
 import { SceneObjectWithUrlSync, SceneTimeRangeState } from './types';
 
 export class SceneTimeRange extends SceneObjectBase<SceneTimeRangeState> implements SceneObjectWithUrlSync {
+  public constructor(state: Partial<SceneTimeRangeState>) {
```

Comment:
I don't like the need to set an empty object to create a `SceneTimeRange`. Why don't we allow creating a scene without arguments?
```suggestion
  public constructor(state: Partial<SceneTimeRangeState> = {}) {
```

Reply:
@ivanortegaalba yea, agree. That is a good idea